src/pages/Flavors.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+import { Container } from 'react-bootstrap';
+
+import FlavorData from '../data/flavors.json';
```

Comment:
Pretty sure our Webpack setup allows you to do `'data/flavors.json';` without the `../` I will test this and include in followup PR if it works.